winfetch.ps1

```diff
@@ -963,22 +966,365 @@ function info_battery {
+        "306" = "Bouvet Island"
+        "307" = "Cayman Islands"
+        "308" = "Channel Islands"
+        "309" = "Christmas Island"
+        "310" = "Clipperton Island"
+        "311" = "Cocos (Keeling) Islands"
+        "312" = "Cook Islands"
+        "313" = "Coral Sea Islands"
+        "314" = "Diego Garcia"
+        "315" = "Falkland Islands"
+        "317" = "French Guiana"
+        "318" = "French Polynesia"
+        "319" = "French Southern Territories"
+        "321" = "Guadeloupe"
+        "322" = "Guam"
+        "323" = "Guantanamo Bay"
+        "324" = "Guernsey"
+        "325" = "Heard Island and McDonald Islands"
+        "326" = "Howland Island"
+        "327" = "Jarvis Island"
+        "328" = "Jersey"
+        "329" = "Kingman Reef"
+        "330" = "Martinique"
+        "331" = "Mayotte"
+        "332" = "Montserrat"
+        "333" = "Netherlands Antilles (Former)"
+        "334" = "New Caledonia"
+        "335" = "Niue"
+        "336" = "Norfolk Island"
+        "337" = "Northern Mariana Islands"
+        "338" = "Palmyra Atoll"
+        "339" = "Pitcairn Islands"
+        "340" = "Rota Island"
+        "341" = "Saipan"
+        "342" = "South Georgia and the South Sandwich Islands"
+        "343" = "St Helena, Ascension and Tristan da Cunha"
+        "346" = "Tinian Island"
+        "347" = "Tokelau"
+        "348" = "Tristan da Cunha"
+        "349" = "Turks and Caicos Islands"
+        "351" = "British Virgin Islands"
+        "352" = "Wallis and Futuna"
+        "742" = "Africa"
+        "2129" = "Asia"
+        "10541" = "Europe"
+        "15126" = "Isle of Man"
+        "19618" = "North Macedonia"
+        "20900" = "Melanesia"
+        "21206" = "Micronesia"
+        "21242" = "Midway Islands"
+        "23581" = "Northern America"
+        "26286" = "Polynesia"
+        "27082" = "Central America"
+        "27114" = "Oceania"
+        "30967" = "Sint Maarten"
+        "31396" = "South America"
+        "31706" = "Saint Martin"
+        "39070" = "World"
+        "42483" = "Western Africa"
+        "42484" = "Middle Africa"
+        "42487" = "Northern Africa"
+        "47590" = "Central Asia"
+        "47599" = "South-Eastern Asia"
+        "47600" = "Eastern Asia"
+        "47603" = "Eastern Africa"
+        "47609" = "Eastern Europe"
+        "47610" = "Southern Europe"
+        "47611" = "Middle East"
+        "47614" = "Southern Asia"
+        "7299303" = "Timor-Leste"
+        "9914689" = "Kosovo"
+        "10026358" = "Americas"
+        "10028789" = "Åland Islands"
+        "10039880" = "Caribbean"
+        "10039882" = "Northern Europe"
+        "10039883" = "Southern Africa"
+        "10210824" = "Western Europe"
+        "10210825" = "Australia and New Zealand"
+        "161832015" = "Saint Barthélemy"
+        "161832256" = "U.S. Minor Outlying Islands"
+        "161832257" = "Latin America and the Caribbean"
+        "161832258" = "Bonaire, Sint Eustatius and Saba"
+    }
+    $languageLookup = @{
```

Comment:
This does not cover all languages (`English (India)` in my case)

Reply:
It seems that the languageLookup array wasn't complete. I'll fix that in my next commit.